server/protocols/protocols-imap4/src/main/java/org/apache/james/imapserver/netty/ReactiveThrottler.java

```diff
@@ -73,7 +81,7 @@ public Mono<Void> throttle(Publisher<Void> task) {
                 String.format(
                     "The IMAP server has reached its maximum capacity "
                         + "(concurrent requests: %d, queue size: %d)",
-                    maxConcurrentRequests, maxQueueSize)));
+                    maxConcurrentRequests, maxQueueSize), imapMessage));
```

Comment:
Missing `%s` for imapMessage

Reply:
No it's not, it's a parameter to the exception, not the string